### ISSUE TITLE
Hashtags and @ handles not separated in E, F, G, H

### ISSUE THREAD

**Issue by mayhewsw** [GateNLP/broad_twitter_corpus#7]
The paper says

> Of note, we separate the preceding symbol in mentions and hashtags (the @ or # characters) as a distinct token, but still include this in entity spans.

According to this schema, sets A and B appear to be tokenized correctly:
```
>> grep -r "@\|#" a.conll
>> grep -r "@\|#" b.conll
```

But E, F, G, and H are not:
```
>> grep -r "@\|#" e.conll
>> grep -r "@\|#" f.conll
>> grep -r "@\|#" g.conll
>> grep -r "@\|#" h.conll
```

**Comment by leondz** [comment]
Thanks. This is because A and B were tokenized differently (historical reasons) - the schema should be unified. We'll put up a fix for this.

**Comment by leondz** [comment]
Fixed in commit ef355b4

**Comment by leondz** [comment]
.. and for CONLL in commit 14d7fe7
